token/program-2022/tests/program_test.rs

```diff
@@ -9,28 +11,40 @@ use {
     std::sync::Arc,
 };
 
-pub struct TestContext {
+pub struct TokenContext {
     pub decimals: u8,
     pub mint_authority: Keypair,
     pub token: Token<ProgramBanksClientProcessTransaction, Keypair>,
     pub alice: Keypair,
     pub bob: Keypair,
-    pub context: Arc<Mutex<ProgramTestContext>>, // ProgramTestContext needs to #[derive(Debug)]
+}
+
+pub struct TestContext {
+    pub context: Arc<Mutex<ProgramTestContext>>,
+    pub token_context: Option<TokenContext>,
 }
 
 impl TestContext {
-    pub async fn new(
-        extension_init_params: Vec<ExtensionInitializationParams>,
-    ) -> TokenResult<Self> {
+    pub async fn new(// extension_init_params: Vec<ExtensionInitializationParams>,
```

Comment:
:eyes:

Reply:
Ew, oops! Cargo fmt trying it's best to handle my slop...